src/tlog-test-timespec.c

```diff
@@ -48,15 +48,16 @@ op_list_test(const struct op *op_list, size_t op_num,
         passed = (res.tv_sec == exp_res.tv_sec && res.tv_nsec == exp_res.tv_nsec);
         if (passed) {
             fprintf(stderr,
-                    "PASS " TLOG_TIMESPEC_FMT " %s " TLOG_TIMESPEC_FMT
+                    "[PASS] " TLOG_TIMESPEC_FMT " %s " TLOG_TIMESPEC_FMT
```

Comment:
Let's not add the `[]' decorations just for one test and together with adding another change. I would prefer to not add them in general, as I think when we need to decorate text, then it means we failed producing a readable output otherwise, and it's better to improve/simplify the output.